src/main.rs

```diff
@@ -309,7 +308,9 @@ mod tests {
                     ("ERROR", "main.rs", "stack backtrace:"),
                     ("0:", "0x", "firecracker::main::"),
                 ],
-            ) || println!("Could not validate backtrace!\n {:?}", Backtrace::new()) != ()
+            ),
+            "Could not validate backtrace!\n {:?}",
```

Comment:
I don't think this works. Check the commit message for why was this workaround introduced: https://github.com/firecracker-microvm/firecracker/commit/73f4eab

Reply:
Okay I've fixed it to a version that maintains behaviour.